magento/adobe-stock-integration
# #768 : [1.1-develop] Introduce MediaGalleryUi module

### Description (*)

This PR will introduce MediaGalleryUi module which will be responsible for Media gallery user interface
### Fixed Issues (if relevant)

1. https://github.com/magento/adobe-stock-integration/issues/768: [1.1-develop] Introduce MediaGalleryUi module

### Manual testing scenarios (*)

1. Login into Magento Backend
2. Check the configuration in in Stores -> Configuration -> Advanced -> System -> Media Gallery fieldset

### Screenshot

![image](https://user-images.githubusercontent.com/39480008/69496692-b9cc2c80-0efa-11ea-977c-4bda56b8421b.png)
